app/helpers/application_helper.rb

```diff
@@ -21,13 +21,13 @@ def current_path_without_query_string
     request.original_fullpath.split("?", 2).first
   end
 
-  def remove_breadcrumbs(content_item)
-    remove_breadcrumbs = false
+  def show_breadcrumbs?(content_item)
+    return false if content_item.nil?
```

Comment:
Is content_item ever `nil`?

Reply:
I don't know, but the original logic included this check so I wasn't confident in removing it.

Reply:
Yes, it can be nil because not all controllers are derived from ContentItemsController (I think there's an argument for a further refactoring where _everything_ has a content_item - some would just be fake - but that's outside the scope of this PR.